plugins/TrimeshReader/plugin.json

```diff
@@ -3,5 +3,5 @@
     "author": "Ultimaker B.V.",
     "version": "1.0.0",
     "description": "Provides support for reading model files.",
-    "api": "6.0.0"
+    "api": "7.0.0"
```

Comment:
To keep it consistent with the others, I'd change this to `7.0`

Reply:
Same as above, 7.0.0 is correct semver, 7.0 (even though we accept it) is technically not.